src/static/js/pluginfw/plugins.js

```diff
@@ -143,15 +143,144 @@ async function loadPlugin(packages, plugin_name, plugins, parts) {
   }
 }
 
+var matcher = function (name, list) {
+  let matches = [];
+
+  try {
+    const nameRegExp = RegExp(name);
+
+    list.forEach(function (item) {
+      if (nameRegExp.test(item)) {
+        matches.push(item);
+      }
+    });
+  } catch(e) {
+    console.log('ERROR', e);
+  }
+
+  return matches;
+}
+
+function conflictMatcher (parts, name, type) {
+  var type;
```

Comment:
`type` already existed and was a function parameter.

This worked because it was erased via the `var`.

```js
function a(x) {
  var x;
  console.log(x);
}

a('hi');
hi
```

With a `let` this would have been caught:
```js
function b(x) {
  let x;
  console.log(x);
}
SyntaxError: Identifier 'x' has already been declared
```